src/bin/stratisd.rs

```diff
@@ -2,7 +2,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-#![allow(dead_code)] // only temporary, until more stuff is filled in
+// only temporary, until more stuff is filled in
```

Comment:
Well, rustfmt did us a favor here. You can drop this commit with PR #209, but now we know that you can still remove the ```#![allow(dead_code)]``` directive and accompanying comment in one of these commits.